aws/elasticache/memcache/outputs.tf

```diff
@@ -2,13 +2,14 @@
  * Cluster outputs
  */
 output "cache_nodes" {
-  value = ["${aws_elasticache_cluster.memcache.cache_nodes}"]
+  value = [aws_elasticache_cluster.memcache.cache_nodes]
```

Comment:
Does this still need the square brackets around it? By the name, `aws_elasticache_cluster.memcache.cache_nodes` looks like it might already be a list, and based on other changes I'm seeing in this PR I wasn't sure if the brackets were still needed here.

Reply:
The 0.12upgrade command doesn't catch everything.  I don't use elasticache so I missed it, too.  cache_nodes is a "list of node objects" so I'd expect the square brackets are not needed.  Do you have an example where this is used so we can confirm it?